### PR TITLE
Update AeroGear Android Push docs

### PR DESCRIPTION
**Pull request by danielpassos**
See: [AGDROID-496](https://issues.jboss.org/browse/AGDROID-496)

### CODE REVIEW

**Comment by danielpassos** [comment]
@secondsun @sebastienblanc @matzew Could you please review

File: docs/unifiedpush/aerogear-push-android/android-app.asciidoc [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
 Let's get started!
 
 === Creating the application
-Fire up Eclipse IDE and choose 'File'-> 'New Android Application'. Fill the 'Application Name' field with the name 'PushApplication' and the 'Package Name' with 'com.push.pushapplication'. As link:http://developer.android.com/google/gcm/gs.html[Google advises], the 'Minimum Required SDK' option for the GCM feature is 'API.8 Android 2.2 (Froyo)' so select this option if not selected. For 'Target SDK' and 'Compile With' options, choose the maximum API SDK installed on your machine.
+Fire up Android Studio IDE and choose 'Start a new Android Studio project'. Fill the 'Application Name' field with the name 'PushApplication' and the 'Package Name' with `com.push.pushapplication`. 
+
+image:./img/android-studio-wizard-1.png[Android Studio - New Project, Wizard (Step 1)]
 
-image:./img/new-project.png[New Project]
+The _Minimum SDK_ option for the link:http://http://github.com/aerogear/aerogear-android-push[AeroGear Android Push] is _API 16 (Android 4.1 - Jelly Bean)_ so select this option if not selected.
```
Comment (matzew) [inline docs/unifiedpush/aerogear-push-android/android-app.asciidoc]: Should we include what version of Androiud Push we are talking about ?

**Comment by matzew** [comment]
@kpeeples mind taking a look here too ? :smile_cat:

File: docs/unifiedpush/aerogear-push-android/android-app.asciidoc (hunk 1 above)
Comment (danielpassos) [inline docs/unifiedpush/aerogear-push-android/android-app.asciidoc]: Done!

**Comment by kpeeples** [comment]
@matzew I will be glad to take a look and provide feedback.

**Comment by secondsun** [comment]
There are still some Eclipse references (http://localhost:4000/docs/unifiedpush/aerogear-push-android/guides/#_eclipse for example)

**Comment by danielpassos** [comment]
@secondsun Nice catch. Fixed, thanks!

**Comment by matzew** [comment]
:+1:

**Pull request merged**